go.sum

```diff
@@ -0,0 +1,12 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
Please remove go.sum

Reply:
Can I just add it to gitignore? It slips in when I forget to remove it